src/FsAutoComplete.Core/CompilerServiceInterface.fs

```diff
@@ -114,24 +114,25 @@ type ParseAndCheckResults
       // TODO: Display other tooltip types, for example for strings or comments where appropriate
       let! tip = checkResults.GetToolTipText(pos.Line, col, lineStr, identIsland, FSharpTokenTag.Identifier)
       let! symbol = checkResults.GetSymbolUseAtLocation(pos.Line, col, lineStr, identIsland)
-      return
```

Comment:
I moved the return so that its at each return point for the match expression, this makes the compiler add sequence points in the pattern match, otherwise I you cant step into any of the code.